dwave/gate/circuit.py

```diff
@@ -12,6 +12,12 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 
+"""Circuit and context manager classes.
+
+Contains classes to construct and handle quantum circuits which contain the quantum operations and
+instructions for running on simulators or hardware. See :py:mod:`dwave.gate.operations` for details
```

Comment:
This is not an easy sentence to parse. The main part is clear, "Contains classes to construct and handle quantum circuits", but the subordinate clause to explain what a circuit is, "contain the quantum operations and instructions for running on simulators or hardware" leaves the verb "running" without a clear subject (running circuits or operations?).

Reply:
I guess it can be sort of both, but what I mean here is running circuits on simulators/hardware. Would this suffice do you think? I can rewrite it otherwise, which might be a good thing to do.
```suggestion
instructions for running these circuits on simulators or hardware. See :py:mod:`dwave.gate.operations` for details
```

Reply:
That solves the problem. Perhaps split into two: "Contains classes to construct and handle quantum circuits. The quantum circuits ..."?